distribution/src/bin/elasticsearch.bat

```diff
@@ -5,6 +5,7 @@ setlocal enableextensions
 
 SET params='%*'
 SET checkpassword=Y
+SET attemptautoconfig=Y
```

Comment:
The default for this in window is always yes, but on linux it looks for an env var. Why the discrepancy?

Reply:
Good catch. I am not sure what @albertzaharovits had in mind with the environment var support but I _don't think_ we need it. I'll wait for him to weigh in before making any changes

Reply:
We don't need to take a default from the env, it was an oversight to leave it there.